discovery-provider/nginx_conf/main.lua

```diff
@@ -35,22 +35,43 @@ end
 function verify_signature (discovery_provider, nonce, signature)
     -- reject if one of the parameter is not provided
     if discovery_provider == nil or nonce == nil or signature == nil then
+        ngx.log(
+            ngx.WARN,
+            "Signature verification failed: ",
+            "discovery_provider=", discovery_provider,
+            ", signature=", signature,
+            ", nonce=", nonce
+        )
         return false
     end
 
     -- reject if nonce was already used
     if ngx.shared.nonce_store:get(discovery_provider .. ";" .. nonce) then
+        ngx.log(
+            ngx.WARN,
+            "Signature verification failed: ",
+            "discovery_provider=", discovery_provider,
+            ", signature=", signature,
+            ", nonce=", nonce
+        )
         return false
     end
 
     -- reject if discovery provider is not in the accept_redirect_from set
-    if not config.accept_redirect_from[discovery_provider] then
-        return false
-    end
+    -- if not config.accept_redirect_from[discovery_provider] then
```

Comment:
recomment?

Reply:
This forces nodes that are running OpenResty to accept requests from this node. As opposed to rejecting requests from other nodes unless it is whitelisted, I think we should only uncomment this when we actually plan to rollout the signature verification to SPs

Reply:
This will also be resolved with the new request counting PR that makes this obsolete, since we will no longer need a list of DPs from an env var.

Reply:
wanna leave an in-line comment?

Reply:
Thanks, added